dbld/builddeps

```diff
@@ -154,22 +152,22 @@ function install_yum_packages {
 }
 
 function install_pip_packages {
-    case "${OS_PLATFORM}" in
+    case "${IMAGE_PLATFORM}" in
         fedora-*)
             python_executables="python"
             ;;
-        ubuntu-focal|tarball)
+        ubuntu-focal|tarball|devshell)
             python_executables="python3"
             ;;
-        devshell|kira)
```

Comment:
This drops python2 from de shell, which I would keep as long as we might have any sensible need for it.

This may be ok to do now, but why keep it in Kira then?

Reply:
Also old versions of debian use python 2 iirc.

Reply:
The reason why we drops python2 from devshell is when I tried run dbld/rules script it throw error, because can't use python2 for install pip, (and also I discussed with the guys, and if I understand everything correctly, devshell shouldn't use the pip at all). 
The Kira need to be keep on python2, because some test function is using only python2

Reply:
Can we clarify this, please?
Why can't you use python2 in devshell? Kira needs it anyway, so if you encountered a issue, it should be fixed in the Kira image too.

Reply:
I re-tried, if devshell still not working with python2, but now it is working with python2. So devshell now using python2 also (as before)

Reply:
That'd good news. Thanks.